sqlite3_opt_dbstat_vtab.go

```diff
@@ -0,0 +1,8 @@
+// +build sqlite_dbstat_vtab dbstat_vtab
```

Comment:
Apply `gofmt` to this file to add modern go:build tag.

Reply:
Ok, fixed